setup.cfg

```diff
@@ -56,6 +56,10 @@ packages = find:
 package_dir =
     =src
 
+[options.extras_require]
```

Comment:
`amplitf` (above) can be removed from the dependencies. Also wonder if other dependencies like `iminuit` can be moved to the `extras_require`. But we had this in ComPWA as well: do we include Minuit (`iminuit`) by default?

Going a bit further: `pandas` is only needed for the `CSVSummary` callback. Could also become optional. So where to draw the line?

Reply:
Note: the pinned dependencies are currently unaffected by `extras_require`, but this will be addressed by @spflueger in a separate PR. https://github.com/ComPWA/tensorwaves/pull/231#discussion_r595892440 can also be addressed there.

Reply:
I think we should define some default packages tensorwaves works with and add those as requirements, so the user can do a complete workflow. Once the user wants extra features some additional packages have to be installed.

Reply:
So that means concretely minuit or scipy is a default (we can just stick with minuit i think). pandas would also be an install requirement if we choose it as a default, is csv summary default? I assume pandas will also be used for some extensions to the data module that might come later on

Reply:
db932dd659f4f383cf464643b0b36c6482fd0563